site/content/3.13/graphs/graph-analytics.md

```diff
@@ -37,11 +40,31 @@ Key features:
 The following lists outlines how you can use Graph Analytics Engines (GAEs).
 How to perform the steps is detailed in the subsequent sections.
 
+{{< tabs "platforms" >}}
```

Comment:
I like this overview! How about adding some links so you can jump straight away to each corresponding section?